timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -72,6 +77,22 @@ template <typename RealType> void declare_neighborlist(py::module &m, const char
 
                 std::vector<std::vector<int>> ixn_list = nblist.get_nblist_host(N, coords.data(), box.data(), cutoff);
 
+                return ixn_list;
+            })
+        .def(
+            "get_nblist_ligand_host",
```

Comment:
I might swap the order of names to hint the intended argument order (`get_nblist_ligand_host` -> `get_nblist_host_ligand`), and I might include an assertion that `K < N` or `K < 1000` or something if warranted. 

Argument order seems consequential in practice: swapping the argument order in [the example from the test](https://github.com/proteneer/timemachine/blob/38c1b1a721fa699a9ae07bd4773126cc7c4da929/tests/test_nblist.py#L191):

```python
host_coords = coords[:num_host_atoms]   # (6282, 3)
ligand_coords = coords[num_host_atoms:] # (34, 3)

# fine
host_ligand_nblist = custom_ops.Neighborlist_f32(num_host_atoms, num_ligand_atoms)
_ = host_ligand_nblist.get_nblist_ligand_host(host_coords, ligand_coords, box, cutoff)

# hangs
ligand_host_nblist = custom_ops.Neighborlist_f32(num_ligand_atoms, num_host_atoms)
_ = ligand_host_nblist.get_nblist_ligand_host(ligand_coords, host_coords, box, cutoff)
```

Reply:
Great catch! Will investigate

Reply:
Oh! This does already hit an assertion.
```
Assertion `N > K' failed.
Aborted (core dumped)
```
 (It just looks like it hangs when executed in a notebook environment...)